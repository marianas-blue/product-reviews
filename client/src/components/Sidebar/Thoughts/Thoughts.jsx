/* eslint-disable react/button-has-type */
/* eslint-disable no-useless-constructor */
/* eslint-disable react/prefer-stateless-function */
import React from 'react';
import { Border, Button, Title, Borderbox } from './Thoughts.style';

class Thoughts extends React.Component {
  constructor(props) {
    super(props);
  }

  render() {
    return (
      <Border>
        <Title>Review this product</Title>
        <Borderbox>Share your thoughts with other customers</Borderbox>

        <Button>
          <a href="#top" style={{ textDecoration: 'none', color: 'black' }}>Write a customer review</a>
        </Button>
      </Border>
    );
  }
}

export default Thoughts;
