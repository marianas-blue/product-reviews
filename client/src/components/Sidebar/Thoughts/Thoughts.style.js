import styled from 'styled-components';

export const Border = styled.div`
border-top: 1px solid #e7e7e7;
border-bottom: 1px solid #e7e7e7;
font-size 14px;
padding: 15px 15px 15px 0px
`;

export const Button = styled.button`
background: #eff1f3;
border-radius: 3px;
border-color: #ADB1B8 #A2A6AC #8D9096;
border-style: solid;
border-width: 1px;
cursor: pointer;
display: inline-block;
margin: 4px;
text-align: center;
text-decoration: none!important;
vertical-align: middle;
box-sizing: border-box;
width: 300px;
height: 29px;
font-family: Lato,sans-serif;
font-size: 13px;
:hover {
  background: #e7e9ec;
}
`;

export const Title = styled.button`
font-weight: bolder;
font-size: 17px;
line-height: 1.255;
border: none; 
padding: 0;
background-color: transparent;
outline: none;
text-align: left;
`;

export const Borderbox = styled.div`
margin-top: 10px;
margin-bottom: 10px;
`;
