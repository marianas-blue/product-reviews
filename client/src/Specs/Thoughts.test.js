/* eslint-disable react/jsx-filename-extension */
/* eslint-disable no-undef */
import React from 'react';
import { shallow } from 'enzyme';

import Thoughts from '../components/Sidebar/Thoughts/Thoughts';

describe('Thoughts', () => {
  it('should render correctly', () => {
    const component = shallow(<Thoughts />);

    expect(component).toMatchSnapshot();
  });
});
