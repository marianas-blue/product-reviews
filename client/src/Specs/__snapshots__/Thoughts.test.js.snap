// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Thoughts should render correctly 1`] = `
<styled.div>
  <styled.button>
    Review this product
  </styled.button>
  <styled.div>
    Share your thoughts with other customers
  </styled.div>
  <styled.button>
    <a
      href="#top"
      style={
        Object {
          "color": "black",
          "textDecoration": "none",
        }
      }
    >
      Write a customer review
    </a>
  </styled.button>
</styled.div>
`;
